draw.go

```diff
@@ -101,16 +101,22 @@ func (m *model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 				m.textAnchorDown()
 			}
 		default:
+			var character = msg.String()
+
+			if strings.HasPrefix(character, "alt+") {
+				character = character[4:]
+			}
```

Comment:
I _think_ you should be able to use `msg.Runes[0]` here:
```suggestion
			if msg.Alt {
				character = msg.Runes[0]
			}
```

Reply:
This gives the following error:

```
# github.com/maaslalani/draw
./draw.go:107:17: cannot use msg.Runes[0] (variable of type rune) as type string in assignment
```

It can be fixed with type casting:
```suggestion
			if msg.Alt {
				character = string(msg.Runes[0])
			}
```
Other than that, it works as expected.